udata_front/theme/gouvfr/assets/js/pages/ReusePublishingForm/Step1DescribeReuse.vue

```diff
@@ -0,0 +1,397 @@
+              </p>
+              <Well class="fr-mt-1w" v-if="fieldHasWarning('image')" color="orange-terre-battue">
+                {{ getWarningText("image") }}
+              </Well>
+            </Accordion>
+          </AccordionGroup>
+      </Sidemenu>
+      <div class="fr-col-12 fr-col-md-7">
+        <Container>
+          <Well
+            color="blue-cumulus"
+            weight="regular"
+            class="fr-mb-2w"
+          >
+            <div class="fr-grid-row">
+              <div class="fr-col-auto fr-mr-3v">
+                <img :src="reuseIcon" alt="" />
+              </div>
+              <div class="fr-col">
+                <p class="fr-m-0 fr-text--bold">{{ t('What is a reuse?') }}</p>
+                <p class="fr-m-0 fr-text--xs">{{ t('A reuse is an exemple of public data\'s usage.') }} {{ t('Publishing a reuse can allow you to gain visibility and start start a dialogue with the dataset producer.') }}</p>
+              </div>
+            </div>
+          </Well>
+          <fieldset class="fr-fieldset" aria-labelledby="description-legend">
+            <legend class="fr-fieldset__legend" id="description-legend">
+              <h2 class="subtitle subtitle--uppercase fr-mb-3v">
+                {{ $t("Producer") }}
+              </h2>
+            </legend>
+            <div class="fr-fieldset__element">
+              <ProducerSelector
+                :user="user"
+                :hasError="fieldHasError('owned')"
+                :errorText="$t('You need to select a Producer')"
+                @update:owned="updateOwned"
+              />
+            </div>
+          </fieldset>
+          <fieldset class="fr-fieldset" aria-labelledby="description-legend">
+            <legend class="fr-fieldset__legend" id="description-legend">
+              <h2 class="subtitle subtitle--uppercase fr-mb-3v">
+                {{ t("Description") }}
+              </h2>
+            </legend>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="nameReuseAccordionId"
+              @blur="vWarning$.title.$touch"
+            >
+              <InputGroup
+                :aria-describedby="nameReuseAccordionId"
+                :label="t('Reuse name')"
+                :required="true"
+                v-model="reuse.title"
+                :hasError="fieldHasError('title')"
+                :hasWarning="fieldHasWarning('title')"
+                :errorText="getErrorText('title')"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="addLinkAccordionId"
+              @blur="vWarning$.url.$touch"
+            >
+              <InputGroup
+                :aria-describedby="addLinkAccordionId"
+                :label="t('Link')"
+                type="url"
+                :placeholder="t('https://...')"
+                :required="true"
+                v-model="reuse.url"
```

Comment:
@maudetes Is there a way to test if an url is already registered as a reuse ? It didn't find any